### DISCUSSION TITLE
What is the name of the theme used in the code examples?

### DISCUSSION THREAD

**Discussion by vdawg-git** [veeso/tui-realm#67] [Q&A] [answered]
![image](https://github.com/veeso/tui-realm/assets/28539403/b1cafc29-aebf-4c41-870c-763585cd0c11)
It looks so good and I want to try it

And thank you for creating this library!

**Comment by veeso** [comment, marked as answer]
Thanks!

If I remember correctly since I don't use it anymore, it should be Horizon.
